.circleci/config.yml

```diff
@@ -9,7 +9,10 @@ jobs:
       - checkout
       - run: apt update && apt install -y ninja-build binutils libtool autoconf automake make cmake gcc g++ pkg-config ragel gtk-doc-tools libfreetype6-dev libglib2.0-dev libcairo2-dev libicu-dev libgraphite2-dev python python-pip
       - run: pip install fonttools
-      - run: ./autogen.sh && (make distcheck || (cd harfbuzz-* && ../.ci/collect-logs.sh && false)) && rm -rf harfbuzz-*
+      - run: ./autogen.sh
+      - run: make
```

Comment:
I think `make` before would be unnecessary or better to test `distcheck` before running a `make`.

Reply:
Plain `make` can fail, but there is no point in trying to show the test logs for such failure.